components/cdc/src/channel.rs

```diff
@@ -207,6 +207,7 @@ pub fn channel(buffer: usize, memory_quota: Arc<MemoryQuota>) -> (Sink, Drain) {
             unbounded_receiver,
             bounded_receiver,
             memory_quota,
+            conn_id: Default::default(),
```

Comment:
If a `conn_id` is needed, try:

```rust
pub fn channel(conn_id: ConnID, buffer: usize, memory_quota: Arc<MemoryQuota>) -> (Sink, Drain)
```

Reply:
The `conn_id` can only be obtained after call `Conn::new(sink, ctx.peer())`. to create the `conn`, we have to create the `sink` first...